inhouse/db_util.py

```diff
@@ -24,7 +24,23 @@ def makePlayer(self, name, id):
             cmd = "INSERT INTO players(id, name, win, loss, ratio, sp) VALUES ('" + str(id) + "', '" + name + "', '0', '0', '0', '500')"
             cur.execute(cmd)
             self.connection.commit()
+            cur.close()
 
+    def addMatchReport(self,id):
```

Comment:
nit on name: `addMatchReporter` seems better to me, this name makes me think that it's a "finished match" report not a user who is allowed to report.